core/src/i18n/en.yaml

```
@@ -988,6 +988,7 @@ jobs:
     at-least-one-container-required: Cron Job must have at least one container.
     container: Container
     containers: Containers
+    containers-readonly-in-edit: Containers are readonly in edit mode.
```

```suggestion
    containers-readonly-in-edit: In edit mode, containers are read-only.
```

Isn't this the opposite of what "edit mode" means? Or can other resources be changed, just not containers? Why do we have to mention edit mode at all? Why not something like:

```suggestion
    containers-readonly-in-edit: You can't edit containers.
 ```
or 
```suggestion
    containers-readonly-in-edit: Containers are read-only.
```

---

You can change containers, but only during resource creation. Should we go with your first comment, or try out something else, like `After a Job is created the containers are read-only.`?

---

Your suggestion explains it best! 
Adding a comma and I don't think Job needs a capital J.

```suggestion
    containers-readonly-in-edit: After a job is created, the containers are read-only.
```